tac-talk-guidelines.md

```diff
@@ -0,0 +1,75 @@
+# Guidelines for Technical Advisory Council (TAC) talks
+
+The Technical Advisory Council (TAC) is part of the Confidential Computing Consortium (CCC), a non-profit organization under the Linux Foundation. The TAC provides a platform for discussing hardware-based trusted execution technologies, their benefits, drawbacks, and implementation approaches. We are a friendly group of a mix of industry representatives, academics, and open-source enthusiasts. The technical background of the audience members is thus a wild mix. Our TAC meetings are attended by: open-source developers that are interested in confidential computing; implementors on any level of the stack that work in industry; academics that write scientific papers about confidential computing on the software or hardwar side; cloud architects that develop solutions for cloud service providers or other large industry players; hardware architects at silicon companies that work on confidential computing features for next generation's products.
+
+Due to this mix, where the only real common ground is an interest in confidential computing, technical talks can be very interesting (and slightly unpredictable) to both the presenter and the audience. When presenting, you need to keep in mind that what you see as obvious in your work might not be known to all members in the audience. That being said, we do appreciate any talk and often enough, the Q&A after the talk ends in interesting debates.
+
+### TL;DR
+- No product pitches, just technical discussions on public topics
+- 20-30 minutes + following Q&A
+- Meetings are recorded and must follow Linux Foundation antitrust policies
+- [LFX meeting calendar](https://confidentialcomputing.io/get-involved/calendar/) for meeting invite
+- An email link to suggest a talk is at the bottom
+
+## Meeting Format
```

Comment:
Maybe condense this with the TL;DR section.. not sure there's much different esp. if the calendar / times are linked per the next comment.

Reply:
Good point, I added the tl;dr at the end when I saw that it was a large document. But I now took the `no advertisements` point into the next list and removed the tl;dr completely.